packages/yew/src/io.rs

```diff
@@ -0,0 +1,98 @@
+//! This module contains types for I/O funtionality.
+
+use std::borrow::Cow;
+
+use futures::channel::mpsc::{self, UnboundedSender};
```

Comment:
> _Me:_ How does `tokio::sync::mpsc` differ from `futures::channel::mpsc`?
> _Eliza:_ the tokio channel has better performance both in time and memory usage
> IMO the biggest and most important difference is that the futures crate's bounded channel will allocate on the heap for every task that's waiting for channel capacity
> this is...Not My Favorite Thing, because it essentially means that if you are using a bounded channel because you want an upper bound on the channel's memory usage...you don't actually get that
> this is not the most fun to be surprised by

From #tokio-users in their discord server. I think we should use the tokio one.

Reply:
`wasm-bindgen-futures` can also be selected for SSR (e.g.: Cloudflare Workers). 

However, tokio channels are as twice fast as futures channels.
I think we should make a compatibility layer for it that bridges to `tokio::sync` or `futures::channel` based on what runtime it is currently using.